hbase-common/src/main/java/org/apache/hadoop/hbase/util/Bytes.java

```diff
@@ -2358,7 +2360,7 @@ public static void zero(byte[] b, int offset, int length) {
     Arrays.fill(b, offset, offset + length, (byte) 0);
   }
 
-  private static final SecureRandom RNG = new SecureRandom();
+  private static final Random RNG = new Random();
```

Comment:
Bytes is marked as IA.Public, so is it OK to change from SecureRandom to normal Random here?
I'm not saying we can not do this, as there is no change on the method signatures, just asking.

Reply:
My interpretation is that private fields are not a part of our IA.Public interface. I approve of this change.

Reply:
Thanks for pointing this out @Apache9 . I think becasue we did not advertise the security of the underlying RNG on Bytes#random (like in javadoc) we are safe here. There should not be an assumption. The new Bytes#secureRandom does imply the underlying RNG is SecureRandom. I think this is an improvement. I will also update the javadoc of these methods to clarify.